lib/config/app-strings.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { addConfigFileNames, getConfigFileNames } from './app-strings';
+import { GlobalConfig } from './global';
+
+describe('config/app-strings', () => {
+  beforeEach(() => {
+    GlobalConfig.reset();
+  });
+
+  it('returns platform specific config filenames if platform detected', () => {
+    GlobalConfig.set({ platform: 'github' });
+    const filenames = getConfigFileNames();
+    expect(filenames.includes('.gitlab/renovate.json')).toBeFalse();
```

Comment:
technically this is a breaking change, not sure if people depend on this 🤔

@rarkins @secustor wdyt?